packages/gasket-plugin-intl/lib/build-modules.js

```diff
@@ -148,19 +155,21 @@ class BuildModules {
   /**
    * Find modules that have /locales folder to process
    *
-   * @returns {string[]} directories - list of paths
+   * @returns {SrcPkgDir[]} source package directories
    */
   async discoverDirs() {
-    const dirs = await fsUtils.getDirectories(this._nodeModulesDir);
+    const pkgDirs = await fsUtils.getPackageDirs(this._nodeModulesDir);
 
-    return dirs.reduce(async (prevPromise, dir) => {
+    return pkgDirs.reduce(async (prevPromise, pkgDir) => {
```

Comment:
This is going to create a lot of in-memory Promises.

You might get better performing making `fsUtils.getPackageDirs` an async generator function, returning an async iterable, and loop through the results with `for await (const [pkgName, dir] of fsUtils.getPackageDirs(this._nodeModulesDir)) {` since you're sequentializing these anyway.

Reply:
This is how it was before, but I can take a stab at it.

Reply:
Ah yes, didn't notice that.